### PR TITLE
chore(project): renaming

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

File: test/aurelia-config.spec.js [hunk 1]
```diff
@@ -1,40 +1,64 @@
 import {Container} from 'aurelia-dependency-injection';
 import {DefaultLoader} from 'aurelia-loader-default';
+import {FrameworkConfiguration, Aurelia} from 'aurelia-framework';
 import {configure} from 'src/aurelia-config';
 import {Config} from 'src/config';
+import extend from 'extend';
```
Comment (RWOverdijk) [inline test/aurelia-config.spec.js]: You don't need `extend`, we have Homefront which exposes a `static .merge()`.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This library is an unofficial plugin for the [Aurelia](http://www.aurelia.io/) p
 ## Overview
 
 Sick of configuring every plugin separately, inconsistent naming and one plugin not knowing what the other does. Here comes the solution. One app config file for all your plugin. It works as follows:
-At aurelia configuration you tell aurelia-config which plugin defaults it should load and the app config objects you want to merge into that sequentially. Then it (optionally) calls sequentially the plugin configurations on them with the (plugin namepsaced) merged settings. 
+At aurelia configuration you tell aurelia-config which plugin config it should load and the app config objects you want to merge into that sequentially. Then it (optionally) calls sequentially the plugin configurations on them with the (plugin namepsaced) merged settings.
```
Comment (RWOverdijk) [inline README.md]: Slightly change the readme. I've written a tiny part to show you what I mean: https://gist.github.com/RWOverdijk/b3a5e875e65d70a05a9788c8774b2e05

File: config.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ System.config({
     "aurelia-logging": "npm:aurelia-logging@1.0.0",
     "aurelia-pal-browser": "npm:aurelia-pal-browser@1.0.0",
     "aurelia-polyfills": "npm:aurelia-polyfills@1.0.0",
+    "extend": "npm:extend@3.0.0",
```
Comment (RWOverdijk) [inline config.js]: Remove, not needed

File: package.json [hunk 4]
```diff
@@ -50,7 +50,8 @@
     "devDependencies": {
       "aurelia-loader-default": "^1.0.0",
       "aurelia-pal-browser": "^1.0.0",
-      "aurelia-polyfills": "^1.0.0"
+      "aurelia-polyfills": "^1.0.0",
+      "extend": "^3.0.0"
```
Comment (RWOverdijk) [inline package.json]: Not needed

File: src/aurelia-config.js [hunk 5]
```diff
@@ -10,10 +10,10 @@ import {Config} from './config';
  * @return {Promise<>}
  */
 export function configure(frameworkConfiguration: FrameworkConfiguration, plugins: Array<string|{}>, ...appConfigs): Promise<> {
-  let config = frameworkConfiguration.container.get(Config);
+  let config        = frameworkConfiguration.container.get(Config);
   let configManager = new ConfigManager(config, frameworkConfiguration.aurelia.loader);
   frameworkConfiguration.container.registerInstance(ConfigManager, configManager);
```
Comment (RWOverdijk) [inline src/aurelia-config.js]: Newline between statements and assignments.

File: src/aurelia-config.js [hunk 6]
```diff
@@ -10,10 +10,10 @@ import {Config} from './config';
...
 
-  return configManager.mergeDefaultsSynchronous(plugins.concat(appConfigs))
-    .then(() => config.fetch('aurelia-config.configure') ? configManager.configurePlugins(frameworkConfiguration, plugins.slice()) : null);
+  return configManager.mergeConfigsSynchronous(plugins.concat(appConfigs))
```
Comment (RWOverdijk) [inline src/aurelia-config.js]:
``` js
return configManager.mergeDefaultsSynchronous(plugins.concat(appConfigs)).then(() => {
  if (config.fetch('aurelia-config.configure')) {
    configManager.configurePlugins(frameworkConfiguration, plugins.slice());
  }
});
```

**Review by RWOverdijk — CHANGES_REQUESTED** [review]
Reviewed. Mostly looks good, but I have some minor feedback.

**Comment by doktordirk** [comment]
@RWOverdijk all new and shiny

**Comment by RWOverdijk** [comment]
I like shiny. I'll review this asap :)

File: test/aurelia-config.spec.js (hunk 1 above)
Comment (doktordirk) [reply test/aurelia-config.spec.js]: done

File: README.md (hunk 2 above)
Comment (doktordirk) [reply README.md]: done

File: config.js (hunk 3 above)
Comment (doktordirk) [reply config.js]: done

File: src/aurelia-config.js (hunk 5 above)
Comment (doktordirk) [reply src/aurelia-config.js]: obsolete/done

File: src/aurelia-config.js (hunk 6 above)
Comment (doktordirk) [reply src/aurelia-config.js]: obsolete

**Review by RWOverdijk — APPROVED** [review]
Did the review two days ago, but I forgot to approve. My bad.

**Comment by doktordirk** [comment]
@RWOverdijk uff :) so please merge (and/or put the repo on sx-elite so i can do it)

**Pull request merged**